src/v/config/configuration.cc

```diff
@@ -85,6 +85,12 @@ configuration::configuration()
       "Maximum amount of bytes to read from one topic read",
       required::no,
       32_KiB)
+  , coproc_script_directory(
```

Comment:
why do we need this? why can't this be derived from the main `redpanda_directory` variable. 

It can simply be `redpanda/coproc/`

Reply:
The least external configurations the better if we can make a smart choice.

Reply:
@senior7515 actually the wasm engine lets this be configurable on its end: https://github.com/vectorizedio/redpanda/blob/dev/src/js/modules/rpc/service.ts#L73

except it stores it under the key 'coproc_engine' in our cfg file. So if I commit this change, the wasm engine needs to look under the redpanda portin of the config file.

In either case, what are your thoughts?